diff --git a/Resources/nginx/nginx.conf b/Resources/nginx/nginx.conf
index a8841fe..8346c0a 100644
--- a/Resources/nginx/nginx.conf
+++ b/Resources/nginx/nginx.conf
@@ -38,7 +38,7 @@ http {
         server app:8000 fail_timeout=0;
     }
 
-    limit_req_zone $binary_remote_addr zone=api:10m rate=2r/s;
+    # limit_req_zone $binary_remote_addr zone=api:10m rate=2r/s;
 
     server {
         listen 80 deferred;
@@ -72,7 +72,7 @@ http {
             add_header Cache-Control "public";
             add_header Pragma public;
 
-            limit_req zone=api burst=10;
+            # limit_req zone=api burst=10;
 
             proxy_set_header X-Real-IP $remote_addr;
             proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
diff --git a/config/settings.py b/config/settings.py
index 1b2731b..44bab3e 100755
--- a/config/settings.py
+++ b/config/settings.py
@@ -161,12 +161,12 @@ REST_FRAMEWORK = {
     'PAGE_SIZE': 20,
 
     'PAGINATE_BY': 20,
-    'DEFAULT_THROTTLE_CLASSES': (
-        'rest_framework.throttling.AnonRateThrottle',
-    ),
-    'DEFAULT_THROTTLE_RATES': {
-        'anon': '1000/hour'
-    }
+#    'DEFAULT_THROTTLE_CLASSES': (
+#        'rest_framework.throttling.AnonRateThrottle',
+#    ),
+#    'DEFAULT_THROTTLE_RATES': {
+#        'anon': '1000/hour'
+#    }
 }
 
 MARKDOWN_DEUX_STYLES = {
